File: LeetCode/Math/Golang/Leetcode-minimum462-movesToEqualArrayElementsII.go
package sbgo

import "sort"

func minMoves2(nums []int) int {
	sort.Ints(nums)
	n, res := len(nums), 0
	var tar = nums[n/2]
	for _, x := range nums {
		if tar > x {
			res += tar - x
		} else {
			res += x - tar
		}
	}
	return res
}

File: LeetCode/Math/JS/Leetcode-minimum462-movesToEqualArrayElementsII.js
var minMoves2 = function (nums) {
    nums.sort((a, b) => a - b);
    let [res, n] = [0, nums.length];
    for (const x of nums) {
        res += Math.abs(x - nums[parseInt(n / 2)]);
    }
    return res;
};